nativelib/src/main/resources/scala-native/gc/none/gc.c

```diff
@@ -52,6 +52,8 @@ size_t scalanative_GC_get_max_heapsize() {
     return Parse_Env_Or_Default("GC_MAXIMUM_HEAP_SIZE", getMemorySize());
 }
 
+size_t scalanative_GC_get_used_heapsize() { return -1L; }
```

Comment:
Is there a way to get the total allocated memory? 
I guess we can add a static variable and explicitly increment it on allocations. But is there a better way perhaps?

Reply:
Allocated memory is not tracked in none GC (it works by always allocating until there is no memory left) Adding a static variable to track allocations should do the job

Reply:
I added the static `TOTAL_ALLOCATED` variable to track the allocated memory.